Add 0.41 runtime to latest updates (#3673)
Qiskit/documentation: docs/guides/latest-updates.mdx

### New Features

*   A new parameter, `tags`, has been added to the [`QiskitRuntimeService`](qiskit-runtime-service "qiskit_ibm_runtime.QiskitRuntimeService") initializer and [`QiskitRuntimeService.save_account()`](qiskit-runtime-service#save_account "qiskit_ibm_runtime.QiskitRuntimeService.save_account") method. `tags` can be used to **filter** instances so only instances with the given tags are returned.

    Additionally, if no valid instances are returned when using the `tags`, `region`, or `plans_preference` filters, an error will now be raised at initialization. Make sure the names of the `tags` and `region` passed in match the instance `tags` or `region` exactly (case-insensitive). For `plans_preference`, as long as a matching plan name is passed in, instances with the matching plan name will be prioritized. ([2277](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2277))

*   Support for running `qiskit-ibm-runtime` with Python 3.9 has been deprecated and will be removed in a future release.

>     Additionally, `qiskit-ibm-runtime` now officialy supports Python 3.13. ([2314](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2314))

Reviewer: ```suggestion
    Additionally, `qiskit-ibm-runtime` now officially supports Python 3.13. ([2314](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2314))
```

> *   There is a new method, [`QiskitRuntimeService.active_instance()`](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service#active_instance "qiskit_ibm_runtime.QiskitRuntimeService.active_instance") which returns the IBM Cloud CRN of the current active instance.

Reviewer: ```suggestion
*   There is a new method, [`QiskitRuntimeService.active_instance()`](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service#active_instance "qiskit_ibm_runtime.QiskitRuntimeService.active_instance"), which returns the IBM Cloud CRN of the current active instance.
```

    The [`QiskitRuntimeService.jobs()`](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service#jobs "qiskit_ibm_runtime.QiskitRuntimeService.jobs") method has been updated to properly accept the `instance` parameter, which can be used to filter jobs. ([2325](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2325))

> *   [`QiskitRuntimeService.usage()`](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service#usage "qiskit_ibm_runtime.QiskitRuntimeService.usage") has been updated to include a `usage_remaining_seconds` field. This field includes the difference between the reported `usage_limit_seconds` or `usage_allocation_seconds` (depending on how the instance is configured) and `usage_consumed_seconds`. Learn more about instance allocation limits [here](/docs/guides/allocation-limits). ([2329](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2329))

Reviewer: ```suggestion
*   [`QiskitRuntimeService.usage()`](/docs/api/qiskit-ibm-runtime/qiskit-runtime-service#usage "qiskit_ibm_runtime.QiskitRuntimeService.usage") has been updated to include a `usage_remaining_seconds` field. This field includes the difference between the reported `usage_limit_seconds` or `usage_allocation_seconds` (depending on how the instance is configured) and `usage_consumed_seconds`. Learn more about instance allocation limits [in the documentation](/docs/guides/allocation-limits). ([2329](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2329))
```

*   Fixed an issue in [`IBMBackend`](/docs/api/qiskit-ibm-runtime/ibm-backend "qiskit_ibm_runtime.IBMBackend") where property changes, such as modifications to `basis_gates`, persisted even after the backend object was renewed. ([2283](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2283))

### Other Notes

*   The deprecated [`IBMBackend`](/docs/api/qiskit-ibm-runtime/ibm-backend "qiskit_ibm_runtime.IBMBackend") attributes, `max_shots` and `max_experiments`, have been removed and the `IBMBackend.max_circuits()` method now returns `None`. See the [job limits guide](/docs/guides/job-limits#job-limits) for details. ([2235](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2235))
*   The deprecated [`IBMBackend`](/docs/api/qiskit-ibm-runtime/ibm-backend "qiskit_ibm_runtime.IBMBackend") and `FakeBackendV2` `defaults()` methods have been removed. They were deprecated in the v0.38.0 release. Pulse defaults have also been removed from all fake backends. ([2238](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2238))

- See the full [0.41.0 release notes.](/docs/api/qiskit-ibm-runtime/release-notes#0400-2025-05-28)

> 

Author: The link here needs to be updated when the `0.41.0` release notes are live
